dimod/discrete/discrete_quadratic_model.py

```diff
@@ -396,7 +491,7 @@ def from_numpy_vectors(cls, case_starts, linear_biases, quadratic,
             if len(labels) != obj._cydqm.num_variables():
                 raise ValueError(
                     "labels does not match the length of the DQM"
-                    )
+                )
```

Comment:
It's fine for this PR but in the future it would be good not to touch unrelated lines of code. It confuses the git history and makes merging harder.

Reply:
I think I should avoid using PyCharm automatic formatting. I'll roll back these changes.